src/webui/api/authcontroller.cpp

```diff
@@ -111,9 +109,9 @@ void AuthController::increaseFailedAttempts()
     FailedLogin &failedLogin = m_clientFailedLogins[sessionManager()->clientId()];
     ++failedLogin.failedAttemptsCount;
 
-    if (failedLogin.failedAttemptsCount == MAX_AUTH_FAILED_ATTEMPTS) {
+    if (failedLogin.failedAttemptsCount >= Preferences::instance()->getWebUIMaxAuthFailCount()) {
```

Comment:
`Preferences::instance()->getWebUIMaxAuthFailCount() > 0` is prerequisite for this method now, IIRC. Please add an assertion.

Reply:
OK, done.